src/main/java/com/sprint/mission/discodeit/controller/UserController.java

```diff
@@ -0,0 +1,85 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.BinaryContentCreateRequest;
+import com.sprint.mission.discodeit.dto.UserCreateRequest;
+import com.sprint.mission.discodeit.dto.UserDto;
+import com.sprint.mission.discodeit.dto.UserUpdateRequest;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Controller
+@RequiredArgsConstructor
+@RequestMapping("/api/users")
+public class UserController {
+    private final UserService userService;
+
+    public BinaryContentCreateRequest resolveProfileRequest(MultipartFile profile) {
+        try {
+            BinaryContentCreateRequest binaryContentCreateRequest = new BinaryContentCreateRequest(
+                    profile.getOriginalFilename(),
+                    profile.getContentType(),
+                    profile.getBytes()
+            );
+            return binaryContentCreateRequest;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @RequestMapping(value = "/findAll", method = RequestMethod.GET)
+    public ResponseEntity<List<UserDto>> findAll() {
+        List<UserDto> users = userService.readAll();
+        return ResponseEntity
+                .status(HttpStatus.OK)
+                .body(users);
+    }
+
+    @RequestMapping(value = "/create", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<User> create(@RequestPart UserCreateRequest userCreateRequest, @RequestPart(required = false) MultipartFile profile) {
```

Comment:
@RequestPart의 value를 지정하지 않으면 포스트맨으로 테스트할 때 아래와 같은 에러가 발생하는 것 같네요.  
```
Request part name for argument type [org.springframework.web.multipart.MultipartFile] not specified, and parameter name information not found in class file either.
```

```suggestion
    public ResponseEntity<User> create(@RequestPart(value = "userCreateRequest") UserCreateRequest userCreateRequest, @RequestPart(value = "profile", required = false) MultipartFile profile) {
```